ui/components/DataTable.tsx

```diff
@@ -561,7 +563,9 @@ function UnstyledDataTable({
                         size="base"
                       />
                       <Spacer padding="xxs" />
-                      <Text color="neutral30">No data</Text>
+                      <Text color="neutral30">
+                        {emptyMessagePlaceholder || "No data"}
+                      </Text>
```

Comment:
Maybe just worth double checking we can add links that look good (preserve their link color etc) as a child element to `<Text />`.. otherwise we could wrap the whole thing?
```suggestion
                      {emptyMessagePlaceholder || (<Text color="neutral30">
                        "No data"
                      </Text>)}
```

Kind of thing?

Reply:
And maybe add `Text` as an export to `index.ts`

Reply:
sound a good idea will commit that